conf/test.config

```diff
@@ -1,28 +1,47 @@
 /*
- * -------------------------------------------------
- *  Nextflow config file for running tests
- * -------------------------------------------------
- * Defines bundled input files and everything required
- * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/rnavar -profile test,<docker/singularity>
- */
+========================================================================================
+    Nextflow config file for running minimal tests
+========================================================================================
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/rnaseq -profile test,<docker/singularity>
+
+----------------------------------------------------------------------------------------
+*/
 
 params {
-  config_profile_name = 'Test profile'
-  config_profile_description = 'Minimal test dataset to check pipeline function'
-  // Limit resources so that this can run on GitHub Actions
-  max_cpus = 2
-  max_memory = 6.GB
-  max_time = 48.h
-
-  // Input data
-  // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-  // TODO nf-core: Give any required params for the test so that command line flags are not needed
-  single_end = false
-  input_paths = [
-    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R1.tiny.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R2.tiny.fastq.gz']],
-    ['SRR389222', ['https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz']]
-  ]
-  // Ignore `--input` as otherwise the parameter validation will throw an error
-  schema_ignore_params = 'genomes,input_paths,input'
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run CI
+    max_cpus   = 2
+    max_memory = 6.GB
+    max_time   = 6.h
+
+    // Input data
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/v3.1/samplesheet_test.csv'
```

Comment:
Make sure that the input for the test matches what we have

Reply:
yes it matched. I have not included single_end flag